pom.xml

```diff
@@ -4035,7 +4035,7 @@
         </property>
       </activation>
       <properties>
-        <hadoop.version>2.9.2</hadoop.version>
+        <hadoop.version>2.10.1</hadoop.version>
```

Comment:
In general, updating a minor version number [major > minor > patch] does not result in compatibility, we just need to compile it successfully locally.
```
mvn clean install -DskipTests -Phadoop-2
```